### PR TITLE
Fix issue when proposed reschedule time is at the past.

### PR DESCRIPTION
**Pull request by NezzarClp**
Now, an error message will be displayed without link to ask for referee for help.

Works by checking now against new_date and throw error accordingly, similar to how other reschedule error is handled.

This is my first pull request and thanks for you time to review in advance xd

### CODE REVIEW

File: backend/common/exceptions.py [hunk 1]
```diff
@@ -140,6 +140,10 @@ class InvalidMpLink(commands.CommandError):
 
     pass
 
+class pastNow(commands.CommandError):
```
Comment (SpartanPlume) [inline backend/common/exceptions.py:143]: The class name should start with a capital letter. Also, `pastNow` more feels like now has been passed (aka future), than the time is in the past, maybe you should rename it to something like `TimeInThePast` ?

File: backend/bot/replies/tosurnament/player.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "not_team_captain": "You're not the team captain of your team, you can't use this command.",
     "parameter": "<match_id> <datetime: MM/dd HH:mm>",
     "contact_referee": "\n\n%2, if you agree to this reschedule, write: `%0anr %3`",
+    "past_now": "Sorry, the requested time is at the past.",
```
Comment (SpartanPlume) [inline backend/bot/replies/tosurnament/player.json:17]: `at` -> `in`

File: backend/bot/modules/tosurnament/player.py [hunk 3]
```diff
@@ -474,6 +476,8 @@ async def reschedule_handler(self, ctx, error):
             await self.send_reply(ctx, ctx.command.name, "past_deadline_end")
         elif isinstance(error, tosurnament.SameDate):
             await self.send_reply(ctx, ctx.command.name, "same_date")
+        elif isinstance(error, tosurnament.ImpossibleReschedule):
```
Comment (SpartanPlume) [inline backend/bot/modules/tosurnament/player.py:479]: this should not check if it is an instance of `ImpossibleReschedule` but of `pastNow`

**Review by SpartanPlume — CHANGES_REQUESTED** [review]
You understood the code so that's good, just some remarks to keep the consistency with the existing code.

**Comment by SpartanPlume** [comment]
Fix #39

File: backend/common/exceptions.py (hunk 1 above)
Comment (NezzarClp) [reply backend/common/exceptions.py]: Chose TimeInThePast as suggested.

File: backend/bot/replies/tosurnament/player.json (hunk 2 above)
Comment (NezzarClp) [reply backend/bot/replies/tosurnament/player.json]: ACK

File: backend/bot/modules/tosurnament/player.py (hunk 3 above)
Comment (NezzarClp) [reply backend/bot/modules/tosurnament/player.py]: Oh my silly mistake... :(

**Comment by NezzarClp** [comment]
x( Still no idea how to make the test works

**Review by SpartanPlume — APPROVED** [review]
Don't worry for the unit test, I'm working on adding them

**Pull request merged**